cves/CVE-2013-0854.yml

```diff
@@ -144,10 +155,12 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
```

Comment:
If there were unit tests, would you need to have in depth knowledge of the system?